documentation/src/main/asciidoc/user_guide/endpoint_interop.adoc

```diff
@@ -1,54 +1,77 @@
 [[endpoint_interop]]
-== Embedded/Remote Interoperability
+== Protocol Interoperability
 
-{brandname} offers the possibility to store and retrieve data in a local embedded way, and also remotely thanks to multiple endpoints offered.
+Clients exchange data with {brandname} through endpoints such as REST or Hot Rod.
 
-When configuring the cache with the link:#encoding_media_type[MediaType] of the data to be stored, users can read and write data in multiple formats taking advantage of the internal support for data conversion.
+Each endpoint uses a different protocol so that clients can read and write data in the cache in a suitable storage format. Because {brandname} can interoperate with multiple clients at the same time, it must convert data between client formats and the storage formats.
 
-Depending on the level of interoperability required of a single cache being accessed from multiple endpoints by different clients, a specific storage format must be configured, clients may need to use different strategies when interacting with cache or even custom classes may be required to be deployed in the server.
+To configure {brandname} endpoint interoperability, you should define the link:#encoding_media_type[MediaType] that sets the format for data stored in the cache.
 
+=== Considerations with Media Types and Endpoint Interoperability
 
-=== Planning for interoperability
+Configuring {brandname} to store data with a specific media type affects client interoperability.
 
-In general REST clients are better suited to deal with text formats such as JSON, XML or plain text instead of binary, although the REST endpoint does support link:#rest_key_content_type[representing binary values in hexadecimal or base64 format].
+REST clients generally handle text formats such as JSON, XML, or plain text better than binary formats. Although the {brandname} REST API does handle binary formats represented as String, link:#rest_key_content_type[encoded in hexadecimal or base64 format].
```

Comment:
Proposal: 
"Although REST clients support sending and receiving link:#rest_key_content_type[encoded binary data], they handle text formats such as JSON, XML, or plain text better than binary formats. "